pkg/dcard/http_client.go

```diff
@@ -0,0 +1,35 @@
+package dcard
+
+import (
+	"fmt"
+
+	"github.com/Dcard/Dcard-Go-SDK"
+)
+
+type HttpClient struct {
+	dcard.Client
+}
+
+type scopeRequest struct {
+	scopes []string
```

Comment:
private 欄位不會出現在 json 裡

```suggestion
	Scopes []string `json:"scopes"`
```